content/develop/tutorials/databases/snowflake.md

```diff
@@ -7,151 +7,220 @@ slug: /develop/tutorials/databases/snowflake
 
 ## Introduction
 
-This guide explains how to securely access a Snowflake database from Streamlit. It uses [st.connection](/develop/api-reference/connections/st.connection), the [Snowpark library](https://docs.snowflake.com/en/developer-guide/snowpark/python/index) and Streamlit's [Secrets management](/develop/concepts/connections/secrets-management). The below example code **will only work on Streamlit version >= 1.28**, when `st.connection` was added.
+This guide explains how to securely access a Snowflake database from Streamlit. It uses [st.connection](/develop/api-reference/connections/st.connection), the [Snowpark library](https://docs.snowflake.com/en/developer-guide/snowpark/python/index) and Streamlit's [Secrets management](/develop/concepts/connections/secrets-management).
+
+### Prerequisites
+
+- The following packages must be installed in your Python environment:
+
+  ```txt
+  streamlit>=1.28
+  snowflake-snowpark-python>=0.9.0
+  snowflake-connector-python>=2.8.0
+  ```
+
+    <Note>
+        Use the correct version of Python required by `snowflake-snowpark-python`. For example, if you use `snowflake-snowpark-python==1.23.0` you must use Python version \>=3.8, \<3.12.
+    </Note>
+
+- You must have a Snowflake account. To create a trial account, see the [tutorial](/get-started/installation/streamlit-in-snowflake) in _Get started_.
+- You should have a basic understanding of [`st.connection`](/develop/api-reference/connections/st.connection) and [Secrets management](/develop/concepts/connections/secrets-management).
 
 ## Create a Snowflake database
 
-<Note>
+If you already have a database that you want to use, you can [skip to the next step](#add-connection-parameters-to-your-local-app-secrets).
 
-If you already have a database that you want to use, feel free to [skip to the next step](#add-username-and-password-to-your-local-app-secrets).
+1. Sign in to your Snowflake account at [https://app.snowflake.com](https://app.snowflake.com).
+1. In the left navigation, click "**Projects**," then click "**Worksheets**."
```

Comment:
select Projects, and then select Worksheets."

If these two items are in the same flow of action, you can write as "select Projects > Worksheets." 
If Projects opens another UI surface on which you select Worksheets, don't write it this way. 

You're not using the ui tags?

Reply:
And you're not using "select"? I don't love "select," but just be consistent.